library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -461,21 +522,31 @@ private boolean purchase(Activity activity, List<String> oldProductIds, String p
 			Bundle bundle;
 			if (oldProductIds != null && purchaseType.equals(Constants.PRODUCT_TYPE_SUBSCRIPTION))
 			{
-				bundle =
-						billingService.getBuyIntentToReplaceSkus(Constants.GOOGLE_API_SUBSCRIPTION_CHANGE_VERSION,
-																 contextPackageName,
-																 oldProductIds,
-																 productId,
-																 purchaseType,
-																 purchasePayload);
+				int apiVersion = Constants.GOOGLE_API_EXTRA_PARAMS_SUPPORTED_VERSION;
```

Comment:
Im worried that this API might not be supported, but same time v5 API (`GOOGLE_API_SUBSCRIPTION_CHANGE_VERSION`) may be. So its kind of a breaking change. We have `isSubscriptionUpdateSupported()` method that was checking if v5 API is supported, but we dont have one to check the availability of this newly introduced API.
So I guess there should be something in this method that checks for API version and uses backward compatible method if needed